src/vaadin-radio-group.ts

```diff
@@ -0,0 +1,22 @@
+import { customElement } from 'lit-element';
+import { RadioGroupBase } from './vaadin-radio-group-base';
+import { RadioGroupMixin } from './vaadin-radio-group-mixin';
+import './vaadin-radio-button';
```

Comment:
If user needs to provide own implementation, it is expected that they would use mixins instead.

Reply:
Yes, but the problem is that if the user provides `<custom-radio-button>` using our mixins but wants to use `<vaadin-radio-group>` "as is" it still imports and registers `<vaadin-radio-button>` too even though it is not used or needed in that case.

Removing this one import line here would make that case more logical to avoid unnecessary import and custom element registration. And I think the only drawback would be that the user needs to always explicitly import `vaadin-radio-button` in addition to `vaadin-radio-group` but I think it's probably ok since it's good practice anyway imo?

Reply:
In case of `custom-radio-button`, rhe `custom-radio-group` is expected to be used too.
We have the same structure of imports for `vaadin-tabs` and `vaadin-tab` also BTW.

Removing import seems like a DX improvement for advanced users.
Keeping it there is a DX improvement for new users (no need to import both).

I would rather optimize for new users in this specific case.